test/conftest.py

```diff
@@ -1099,18 +1088,6 @@ def get_document_store(
             knn_engine="faiss",
         )
 
-    elif document_store_type == "opensearch":
-        document_store = OpenSearchDocumentStore(
-            index=index,
-            return_embedding=True,
-            embedding_dim=embedding_dim,
-            embedding_field=embedding_field,
-            similarity=similarity,
-            recreate_index=recreate_index,
-            port=9201,
-            knn_engine="nmslib",
-        )
-
```

Comment:
We have `test_opensearch.py`, and we are removing OpenSearchDocumentStore from `get_document_store` yet we don't do the same for `ElasticsearchDocumentStore`. We are not removing it from `get_document_store` and we added 'test_elasticsearch.py'. Just a bit confused here. What's happening?